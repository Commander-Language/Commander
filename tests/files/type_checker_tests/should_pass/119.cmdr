//Test that greater than or equal operation type checks
x = "hi" >= "there";
