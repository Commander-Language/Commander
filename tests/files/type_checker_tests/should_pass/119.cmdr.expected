(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode x) SET (BinOpExprNode (StringExpressionNode (StringNode (StringExprsNode (StringExpressionNode (StringNode 'hi') STRING))) STRING) GREATER_EQUAL (StringExpressionNode (StringNode (StringExprsNode (StringExpressionNode (StringNode 'there') STRING))) STRING) BOOL) BOOL)))
